riot/riot.py

```diff
@@ -32,15 +31,29 @@ def __init__(self, *args, **kwargs):
         self.__dict__ = self
 
 
+def rm_singletons(d: t.Dict[_K, t.Union[_V, t.List[_V]]]) -> t.Dict[_K, t.List[_V]]:
+    return {k: to_list(v) for k, v in d.items()}
+
+
+def to_list(x: t.Union[_K, t.List[_K]]) -> t.List[_K]:
+    return [x] if not isinstance(x, list) else x
```

Comment:
I'm approving this but having 2 simple unit tests for those functions could be worth the 5 minutes.
Feel free to add them later.

Reply:
seems like a good place for some doctests!

Reply:
Done in https://github.com/DataDog/riot/pull/77